moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -509,42 +392,8 @@ const ProjectComponentEdit = ({
    * Persists the changes to the database
    */
   const handleSaveButtonClick = () => {
-    // Retrieve current project component id
-    const projComponentId = getProjectComponentId();
-
     // First, we need a list of map changes only
-    const mapListOfChanges = generateMapUpserts();
-
-    // For each change, we generate a list of feature component objects
-    const featureComponents = mapListOfChanges.map(feature => ({
```

Comment:
this chunk of code took the previously identified geojson features we needed to update and converted them into graphql objects. thanks to simplified schema it is covered by the above `generateFeatureUpserts`